### DISCUSSION TITLE
Getting ERROR `/_bridgetown/static/js/autoall.<webpack output just cerated>.js' not found.

### DISCUSSION THREAD

**Discussion by GumptionWare** [bridgetownrb/bridgetown#237] [Q&A] [answered]
Webpack seems to be doing OK with a fresh `yarn start`:
`yarn start`
`yarn run v1.22.10`
`$ node start.js`
`$ webpack --mode development -w`
`$ bundle exec bridgetown serve --port 4001`
`[Webpack] assets by status 3.55 KiB [cached] 1 asset`
`[Webpack] asset ../css/all.5704d52f1c88442b5c31.css 13.4 KiB [emitted] [immutable] (name: main) 1 related asset`
`[Webpack] asset ../../../../.bridgetown-webpack/manifest.json 228 bytes [emitted] [compared for emit]`
`[Webpack] Entrypoint main 17 KiB (7.47 KiB) = ../css/all.5704d52f1c88442b5c31.css 13.4 KiB all.424312202dbb9dce9cb4.js 3.55 KiB 2 auxiliary assets`
[`Webpack] webpack 5.21.1 compiled successfully`
`[Bridgetown] Configuration file: /Users/jeffhelman/Dropbox/CONTENT/SpendAbility/SpendableContent/bridgetown.config.yml`
`[Bridgetown]           Starting: Bridgetown v0.19.2 (codename "Arbor Lodge")`
`[Bridgetown]        Environment: development`

... but when Browsersync (I think) tries to do its thing, I am consistently seeing these errors:

`[Live] [Browsersync] Watching files...`
`[Bridgetown] [2021-02-07 11:22:00] ERROR `/_bridgetown/static/js/autoall.424312202dbb9dce9cb4.js' not found.`

Does this point to something I am doing wrong with a configuration setting?

**Comment by ayushn21** [comment, marked as answer]
Hey @GumptionWare, we don't support Webpack 5 as yet in Bridgetown sadly. There's an issue open for it (https://github.com/bridgetownrb/bridgetown/issues/204). The issue details a workaround if you're keen to upgrade :). We'll try to resolve the issue soon!

**Reply by GumptionWare** [reply]
That is so ironically funny! I jumped through many hoops to upgrade to V5 because I was getting other errors that pointed to needing that. I'll check that issue out. Many thanks, @ayushn21!

**Reply by ayushn21** [reply]
No worries! Let me know if you run into any issues. :)
